### ISSUE TITLE
sportran-analysis fails without "-w <W>"

### ISSUE THREAD

**Issue by bernstei** [lorisercole/thermocepstrum#66]
If I try to run `sportran-analysis` without a `-w` flag, I get the following error
```
> sportran-analysis --param-from-input-file-column Temp TEMPERATURE --param-from-input-file-column Volume VOLUME -k flux -t 5 --input-format table -u metal -r -c 2 --TSKIP 2 --output TCEPS.out -- tmp2.txt
.
.
.
Traceback (most recent call last):
  File "/home/cluster2/bernstei/.local/bin/sportran-analysis", line 8, in <module>
    sys.exit(main())
  File "/home/cluster2/bernstei/.local/lib/python3.9/site-packages/sportran/analysis.py", line 210, in main
    run_analysis(args)
  File "/home/cluster2/bernstei/.local/lib/python3.9/site-packages/sportran/analysis.py", line 476, in run_analysis
    outarray = np.c_[j.freqs_THz, j.psd, j.fpsd, j.logpsd, j.flogpsd]
  File "/home/Software/python/system/extra/lib64/python3.9/site-packages/numpy/lib/index_tricks.py", line 412, in __getitem__
    res = self.concatenate(tuple(objs), axis=axis)
  File "<__array_function__ internals>", line 180, in concatenate
ValueError: all the input array dimensions for the concatenation axis must match exactly, but along dimension 0, the array at index 0 has size 5 and the array at index 2 has size 1
```
From what I can tell the issue is that two of the arrays being passed to `np.c_`, `j.fpsd` and `j.flogpsd`, are `None`, so the concatenation fails.

If I add a `-w 1`, which sets a filter width and therefore creates `j.fpsd`, it doesn't fail here.  Note that with this example input it does fail later, but that's just because my example file is _very_ short.  With a reasonable length input it succeeds if and only if I pass a `-w` argument.

The example input file is attached.

[tmp2.txt](https://github.com/sissaschool/sportran/files/10116077/tmp2.txt)

**Comment by rikigigi** [comment]
Pull request #67 should fix that issue. Can you check, please?

**Comment by lorisercole** [comment]
Fixed by #67
